uiowa/uiowa
# ITS Permissions Update

Resolves https://github.com/uiowa/uiowa/issues/7456

# To Test

```
ddev blt ds --site=its.uiowa.edu
```

- Create and masquerade as an editor on the website.
- You should be able to create/edit/delete support articles and toggle the published state (unpublished by default).
- Because categories was switched to radios, we don't want a N/A choice so I made it required.
- Role mapping recommendations doc: https://github.com/uiowa/uiowa/issues/7456#issuecomment-2015891103
